.vib/common/goss/templates/check-php-fpm.yaml

```diff
@@ -0,0 +1,41 @@
+file:
+  /opt/bitnami/php/etc/php.ini:
+    exists: true
+    contains:
+      - /^opcache.interned_strings_buffer = 16/
+      - /^opcache.memory_consumption = 192/
+      - /^opcache.file_cache = \/opt\/bitnami\/php\/var\/run\/opcache_file/
+      - /^upload_tmp_dir = \/opt\/bitnami\/php\/tmp/
+      - /^session.save_path = \/opt\/bitnami\/php\/var\/run\/session/
+      - /^error_log = \/dev\/stderr/
+  /opt/bitnami/php/etc/php-fpm.d/www.conf:
+    exists: true
+    contains:
+      - /^listen = \/opt\/bitnami\/php\/var\/run\/www.sock/
+  /opt/bitnami/php/logs/php-fpm.log:
+    filetype: symlink
+    exists: true
```

Comment:
Nitpick: we normally add the `linked-to` attribute, though I don't have any strong thoughts about it.

Reply:
Nice tip! I didn't know about that attribute.

Updated the test adding it.